loglist2/logfilter_test.go

```diff
@@ -208,10 +208,74 @@ func TestTemporalCompatible(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			test.cert.NotBefore = test.notBefore
-			got := test.in.TemporalCompatible(test.cert)
+			got := test.in.TemporallyCompatible(test.cert)
 			if diff := pretty.Compare(test.want, got); diff != "" {
 				t.Errorf("Getting NotBefore-compatible logs diff: (-want +got)\n%s", diff)
 			}
 		})
 	}
 }
+
+func TestCompatible(t *testing.T) {
+	cert, _ := x509util.CertificateFromPEM([]byte(testdata.TestPreCertPEM))
+	caCert, _ := x509util.CertificateFromPEM([]byte(testdata.CACertPEM))
+
+	tests := []struct {
+		name      string
+		in        LogList
+		notBefore time.Time
+		cert      *x509.Certificate
+		rootCert  *x509.Certificate
+		roots     LogRoots
+		want      LogList
+	}{
+		{
+			name:      "RootedChain",
+			in:        sampleLogList,
+			notBefore: stripErr(time.Parse(time.UnixDate, "Sat Mar 8 11:06:00 PST 2016")),
```

Comment:
nit: Could use mustParseTime() here.

Reply:
Seems got 2 equivalent func for that. Clean up to follow.